components/helper.tsx

```diff
@@ -0,0 +1,27 @@
+import { useState, useEffect } from "react";
+
+// RECOMMENDED NOT TO USE
```

Comment:
it says recommended not to use, and i don't think we're actually using it?
Is it for something future / in progress? or can we just remove this and all references to it?

Reply:
removed in new PR